configs/components/facter.rb

```diff
@@ -197,11 +197,6 @@
 
   pkg.install_file ".gemspec", "#{settings[:gem_home]}/specifications/#{pkg.get_name}.gemspec"
 
-  if platform.is_windows?
-    pkg.directory File.join(settings[:sysconfdir], "facter", "facts.d")
-  else
-    pkg.link "#{settings[:bindir]}/facter", "#{settings[:link_bindir]}/facter"
-    pkg.directory File.join('/opt/puppetlabs', 'facter')
```

Comment:
Oh, but the two with `pkg.directory File.join('/opt/puppetlabs', 'facter')` and `pkg.directory File.join('/opt/puppetlabs', 'facter', 'facts.d')` were unnecessarily redundant and `pkg.link "#{settings[:bindir]}/facter", "#{settings[:link_bindir]}/facter"` is still getting called on non-windows